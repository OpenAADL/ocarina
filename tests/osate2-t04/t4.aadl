package t4
public
with other_package_name;
system system_t4
end system_t4;
end t4;