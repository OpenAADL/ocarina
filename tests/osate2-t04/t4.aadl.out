t4.aadl:3:06: other_package_name (identifier) is not a package or a property set visible or existing
Cannot analyze AADL specifications
